src/packages/sdldoom-1.10/f_finale.c

```diff
@@ -716,8 +724,15 @@ void F_Drawer (void)
 	      V_DrawPatch (0,0,0,
 			 W_CacheLumpName("CREDIT",PU_CACHE));
 	    else
-	      V_DrawPatch (0,0,0,
-			 W_CacheLumpName("HELP2",PU_CACHE));
+	    {
+	      // Check if HELP2 exists, use CREDIT as fallback
```

Comment:
Fix for game always crashing when it should end.
This doom wad file does not have the Help2 screen that would otherwise be shown.
I think it's just a screen telling you to buy full version or something.